tests/sqlparser_bigquery.rs

```diff
@@ -845,10 +845,10 @@ fn parse_like() {
 
         // Test with escape char
         let sql = &format!(
-            "SELECT * FROM customers WHERE name {}LIKE '%a' ESCAPE '\\'",
```

Comment:
without `r#"...` the string was in the end containing `.. ESCAPE '\'`, what I think is invalid character see

![CleanShot 2024-02-25 at 12 34 14@2x](https://github.com/getsynq/sqlparser-rs/assets/143357/bdf88733-1e9e-4558-98a5-dcebffe494d4)